bitnami/kafka/README.md

```diff
@@ -385,6 +387,37 @@ It is strongly recommended to use immutable tags in a production environment. Th
 
 Bitnami will release a new chart updating its containers if a new version of the main container, significant changes, or critical vulnerabilities exist.
 
+### Datadog Autodiscovery Integration
```

Comment:
Can you add here a paragraph with a basic explanation apart from the code block?